extension.js

```diff
@@ -22,15 +22,23 @@ let grab_window = null;
 const APP_ID = 'com.github.amezin.ddterm';
 const APP_DBUS_PATH = '/com/github/amezin/ddterm';
 const WINDOW_PATH_PREFIX = `${APP_DBUS_PATH}/window/`;
-const SUBPROCESS_ARGV = [Me.dir.get_child('com.github.amezin.ddterm').get_path(), '--undecorated'];
+const SUBPROCESS_ARGV = [
+    Me.dir.get_child('com.github.amezin.ddterm').get_path(),
+    '--undecorated',
+];
 const IS_WAYLAND_COMPOSITOR = Meta.is_wayland_compositor();
 const USE_WAYLAND_CLIENT = Meta.WaylandClient && IS_WAYLAND_COMPOSITOR;
 const SIGINT = 2;
 
 class ExtensionDBusInterface {
     constructor() {
-        let [_, xml] = Me.dir.get_child('com.github.amezin.ddterm.Extension.xml').load_contents(null);
-        this.dbus = Gio.DBusExportedObject.wrapJSObject(ByteArray.toString(xml), this);
+        let [_, xml] = Me.dir
+      .get_child('com.github.amezin.ddterm.Extension.xml')
+      .load_contents(null);
+        this.dbus = Gio.DBusExportedObject.wrapJSObject(
+            ByteArray.toString(xml),
+            this
+        );
```

Comment:
Why do you reformat the code in a way that obviously doesn't match the rest of the file? And why do you reformat the code that you didn't need to modify at all?